boilerplate/fullstack/remix/app/routes/auth.$.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import SuperTokensUI, { canHandleRoute } from "supertokens-auth-react/ui/index.js";
+import { PreBuiltUIList } from "../../app/config/frontend";
+import SuperTokens from "supertokens-auth-react/index.js";
+
+export default function Auth() {
+    useEffect(() => {
+        if (canHandleRoute(PreBuiltUIList) === false) {
+            SuperTokens.redirectToAuth({
+                redirectBack: false,
+            });
+        }
+    }, []);
```

Comment:
remove this cause there is already a useEffect that does this in this file

Reply:
✔️